modules/installation-vsphere-machines.adoc

```diff
@@ -14,29 +14,25 @@ To install {product-title} on user-provisioned infrastructure on VMware vSphere,
 
 .Prerequisites
 
-* Obtain the Ignition config files for your cluster.
-* Create a link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.vcenterhost.doc/GUID-B1018F28-3F14-4DFE-9B4B-F48BBDB72C10.html[vSphere cluster].
+* You have obtained the Ignition config files for your cluster.
+* You have created a link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.vcenterhost.doc/GUID-B1018F28-3F14-4DFE-9B4B-F48BBDB72C10.html[vSphere cluster].
+* You have installed the VMWare PowerCLI (`vm`) command.
 
 .Procedure
 
-. Convert the control plane, compute, and bootstrap Ignition config files to Base64 encoding.
-
-+
-For example, if you use a Linux operating system, you can use the `base64` command to encode the files.
+. Locate the following Ignition config files that the installation program created, for example:
 +
-[source,terminal]
-----
-$ base64 -w0 <installation_directory>/master.ign > <installation_directory>/master.64
-----
+* `<installation_directory>/bootstrap.ign`
+* `<installation_directory>/master.ign`
+* `<installation_directory>/worker.ign`
 +
-[source,terminal]
-----
-$ base64 -w0 <installation_directory>/worker.ign > <installation_directory>/worker.64
-----
+. Convert the Ignition config files to Base64 encoding and store them in the extra configuration properties of the virtual machine (VM) before booting by running the PowerCLI (`vm`) command:
 +
 [source,terminal]
 ----
-$ base64 -w0 <installation_directory>/bootstrap.ign > <installation_directory>/bootstrap.64
+$ vm | New-AdvancedSetting \
+-name "guestinfo.ignition.config.data" \
+-value $Ignition -confirm:$false -Force > $null
```

Comment:
Hi @jcpowermac - I've added the PowerCLI command here per your last comment. But I'm not clear as to whether this can be run once for all Ignition config files, or if it needs to be run individually for bootstrap, master, and worker Ignition config files? Could you review and let me know what's missing?

Here's the latest build preview: http://file.rdu.redhat.com/bfuru/0820/restore-vsphere-ipi-steps/installing/installing_vmc/installing-restricted-networks-vmc-user-infra.html#installation-vsphere-machines_installing-restricted-networks-vmc-user-infra

Thanks!